include/zephyr/linker/common-ram-end.ld

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
IMHO, the name of this file, `common-ram-end.ld`, can be somewhat misleading because we have `common-ram.ld` and there is no direct association between these two files.

To elaborate, `common-ram-end.ld` sounds like it is supposed to be included by `common-ram.ld` at the end, or by one's own linker script after `common-ram.ld`, which is not the intended use of this file.

Since this file is meant to be included at the end of RAM (and not just "common RAM"), should we rename it to `ram-end.ld`?

Reply:
I thought the "common" prefix was to indicate that it could be used from any linker script setup? In any case, naming is hard. Happy to change to whatever seems reasonable.

Reply:
> I thought the "common" prefix was to indicate that it could be used from any linker script setup?

No. For example, `debug-sections.ld` and `snippets-sections.ld` do not have the "common" prefix.

"common" in "common-ram" means it defines the RAM sections that are common to all archs/SoCs.

Reply:
sounds good. I've renamed common-ram-end.ld and common-ram-end.cmake